ray-operator/config/samples/ray-cluster.kube-scheduler.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  name: test-podgroup-0
+  labels:
+    ray.io/gang-scheduling-enabled: "true"
```

Comment:
I don't think we should be using ray.io/* labels for any of the scheduler integrations. The label prefix should be specific to the integration (see other examples with Volcano, Yunikorn and Kueue)